controllers/sandbox_controller_test.go

```diff
@@ -188,6 +189,7 @@ func TestReconcilePod(t *testing.T) {
 		initialObjs []runtime.Object
 		sandbox     *sandboxv1alpha1.Sandbox
 		wantPod     *corev1.Pod
```

Comment:
I do think we could do this quite concisely with a yaml format golden test. Not a blocker